package.json

```diff
@@ -91,7 +111,11 @@
         "tslib": "^1.10.0",
         "tslint": "^5.19.0",
         "typedoc": "^0.15.0",
-        "typescript": "3.2.4"
+        "typescript": "3.2.4",
+        "@ngrx/store-devtools": "7.2.0",
+        "ngrx-store-freeze": "0.2.4",
```

Comment:
Hi @fkolar Do we need ngrx there ?

Reply:
No we will be able to remove this is not needed. It is jsut default behavior when you add nx schematics. But this is minor, we can remove this once, we are happy with this structure and I  can do 1 quick commit.

Reply:
I went deeply into the, in case we want to have more, libs/docs it will be great, also the fact that we have paths shortcut is perfect. I'm happy with this structure so far, let's wait for other teammates opinion.

Reply:
I briefly run this structure with deno on one of the call, but yes we want to createanother layer on top of this called @xxxx/platform, to provide more simple and closed API , turing directives into components, introducing a form concept, adn some other thigns , plus the main motivation behind this is mainly this `@component` so we can instantiate things programatically. 

You guys, Inna, deno need to review this once more, to make sure existing functionality works as before, mainly CI, publishing etc.. 
Then we will have add manually all the commits  comes after 04/09/2019